.gitignore

```diff
@@ -1,18 +1,3 @@
-# Build and Release Folders
-bin-debug/
-bin-release/
-[Oo]bj/
-[Bb]in/
-
-# Other files and folders
-.settings/
-
-# Executables
-*.swf
-*.air
-*.ipa
-*.apk
-
-# Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
-# should NOT be excluded as they contain compiler settings and other important
-# information for Eclipse / Flash Builder.
+# .gitignore
```

Comment:
- [ ] Great work. According to the guidelines, your .gitignore file should list `test.md` file and `node_modules` directory (as it can be added by accident while setting up Linters). Please kindly add the test.md